share/revng/test/configuration/revng-c/simplify-switch.yml

```diff
@@ -15,5 +15,5 @@ commands:
       revng artifact
         --resume "$OUTPUT" --analyze simplify-switch
         --abi-enforcement-level=no "$INPUT" -o /dev/null ;
-      FileCheck
-        --input-file="$OUTPUT/simplify-switch/module.ll" "${SOURCE}.filecheck";
+      zstdcat "$OUTPUT/simplify-switch/module.ll" | llvm-dis |
```

Comment:
As a convention, we prefer `opt -S` to `llvm-dis`.

Reply:
Done